light-prover/main.go

```diff
@@ -791,10 +791,6 @@ func getKeysByArgs(context *cli.Context, isTestMode bool) ([]string, error) {
 		circuitTypes = append(circuitTypes, prover.Combined)
 	}
 
-	if !inclusion && !nonInclusion {
-		return nil, fmt.Errorf("no circuit type provided")
-	}
```

Comment:
this prevented the prover from starting with only append or update circuits.

Reply:
fix https://github.com/Lightprotocol/light-protocol/pull/1305